pkg/storage/s3/s3.go

```diff
@@ -106,14 +97,26 @@ func (d *driver) createBucket(svc *s3.S3) error {
 }
 
 func (d *driver) CompleteConfiguration() error {
+	s3cfg, err := util.GetConfig(d.Name, d.Namespace)
```

Comment:
get this directly from the install-config object.

Reply:
updated in next push